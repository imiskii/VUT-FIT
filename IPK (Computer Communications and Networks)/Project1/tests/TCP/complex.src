HELLO
SOLVE (+ (* 10 10) (* 10 10))
SOLVE (* 1000000 1000000)
SOLVE (+ 5 6.58)
SOLVE (/ 65 8)
BYE