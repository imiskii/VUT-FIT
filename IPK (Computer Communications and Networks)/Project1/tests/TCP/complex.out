HELLO
RESULT 200
RESULT 1000000000000
RESULT 11.58
RESULT 8.125
BYE

